docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+# Product Overview
+
+## OpenTripPlanner project
+
+OpenTripPlanner is a group of open source software applications that help individuals and organizations 
+identify and digitize multimodal trip plans based on OpenStreetMap (OSM) and other standardized data 
+sources (e.g. GTFS, GBFS, NeTEx).
+
+OpenTripPlanner is a community of dozens of individuals and organizations that work collaboratively to 
+improve multimodal trip planning best practices and to make it easier for public transit agencies and 
+public transit riders to publish and access information about transit services.
+
+OpenTripPlanner is locally managed in as many different ways as there are OpenTripPlanners projects. 
```

Comment:
My sense is there's only one "OpenTripPlanner project" and the software produced by that project is deployed or used in many local "instances" or "deployments", or "OpenTripPlanner-based systems". Saying OTP itself is "locally managed" could give imply that there are many diverging forks of the software.

Reply:
New wording: "OpenTripPlanner deployments are locally managed in many different ways by many different types of organizations."